internal/testutil/test_envar_keys.go

```diff
@@ -0,0 +1,175 @@
+package testutil
+
+//########################################
+//###### PORTS / CROSS CONNECT / VC
+//########################################
+
+// PORTS
+const PF_PORT_MEDIA_KEY = "PF_PORT_MEDIA"
+const PF_PORT_POP1_KEY = "PF_PORT_POP1"
+const PF_PORT_AVZONE1_KEY = "PF_PORT_AVZONE1"
+const PF_PORT_POP2_KEY = "PF_PORT_POP2"
+const PF_PORT_AVZONE2_KEY = "PF_PORT_AVZONE2"
+const PF_PORT_SUBTERM_KEY = "PF_PORT_SUBTERM"
+const PF_PORT_AUTONEG_KEY = "PF_PORT_AUTONEG"
+const PF_PORT_SPEED_KEY = "PF_PORT_SPEED"
```

Comment:
I think it is safe to assume that we want to run tests with the lowest possible speed ($$$), so probably this var would be optional.

Reply:
That's a good point, we can find ways to accomplish that without hardcoded code.

Reply:
Resolving this conversation. Thanks for the comments guys.